### ISSUE TITLE
vsfeedback -- Incorrect target framework installed 

### ISSUE THREAD

**Issue by rrelyea** [NuGet/Home#4390]
filed by @dtivel in vsfeedback 745993

Scenario 31

Repro steps:

1.  Open the solution in the attached "ClassLibrary (2).zip" file.
2.  Set a breakpoint in ConsoleAppNetFx\Program.cs.
3.  Start debugging the ConsoleAppNetFx project and verify that the "message" value is "Hello from .NET Framework running Microsoft Windows NT 6.2.9200.0"
4.  Stop debugging.
5.  Remove the ClassLibrary project reference from the ConsoleAppNetFx project.
6.  In File Explorer, copy the path of the directory that contains the ClassLibrary.1.0.0.nupkg package.
7.  In Visual Studio navigate on the main menu to Tools | NuGet Package Manager | Package Manager Settings 
8.  In the subsequent Options dialog, in the left-hand tree pane click on Package Sources.
9.  In the right-hand details pane, click the green plus button to create a new package source.
10.  At the bottom of the dialog paste the copied package folder path into the Source text box.
11.  Click Update.
12.  Click OK.
13.  In Solution Explorer right-click on ConsoleAppNetFx | References and select "Manage NuGet Packages..."
14.  In the subsequent NuGet Package Manager UI change the "Package source" dropdown in the upper-right-hand corner to the newly created package source ("Package source" by default).
15.  In the Browse tab select the ClassLibrary package.
16.  In the right-hand details pane click the Install button.
17.  In the subsequent "Choose NuGet Package Manager Format" dialog, select "PackageReference in project file" and click OK.
18.  In the subsequent "Preview changes" dialog click OK.
19.  Start debugging the ConsoleAppNetFx project and inspect the "message" value.

Expected result:

"message" is "Hello from .NET Framework running Microsoft Windows NT 6.2.9200.0"

Actual result:

"message" is "Hello from .NET Standard"

Observation:

The ConsoleAppNetFx project.assets file indicates that the netstandard1.4 target framework was installed instead of the net461 target framework.  The ConsoleAppNetFx project targets .NET Framework 4.6.1.

**Comment by emgarten** [comment]
The nupkg created with ``GeneratePackageOnBuild`` only contains 1/2 TFMS. 

@rohit21agrawal was this fixed by https://github.com/NuGet/Home/issues/4381?

### csproj:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFrameworks>net461;netstandard1.4</TargetFrameworks>
    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
  </PropertyGroup>
</Project>
```

### Resulting nupkg
``ClassLibrary.1.0.0\lib\netstandard1.4\ClassLibrary.dll``

No other lib files found.

**Comment by rohit21agrawal** [comment]
@emgarten yes this was fixed by #4381 , however, when @dtivel filed this issue, i did check the nupkg on his machine, and it did have both the TFMs in the lib folder

**Comment by dtivel** [comment]
Does not repro on latest builds with additional private fixes (NuGet/NuGet.Client#1159 and NuGet/NuGet.Client#1148).
